app/controllers/applications_controller.rb

```diff
@@ -21,4 +19,20 @@ def create
       flash[:alert] = "Error: #{error_message(@application.errors)}"
     end
   end
+
+  def update
```

Comment:
Good to know the discrepancy between update and create, great application of update here.